### PR TITLE
Ome k8s lochy

### PR DESCRIPTION
**Pull request by kennethgillen**
Building on https://github.com/openmicroscopy/prod-playbooks/pull/39 referencing org's tag rather than personal branch.

### CODE REVIEW

**Comment by joshmoore** [comment]
Re-opening now that gh-42 is merged

File: requirements.yml [hunk 1]
```diff
@@ -6,8 +6,9 @@
 - src: openmicroscopy.lvm-partition
   version: 1.1.0
 
-- src: openmicroscopy.network
-  version: 1.0.0
+- name: openmicroscopy.network
+  src: 1.1.0
```
Comment (sbesson) [inline requirements.yml]: This causes 

```
- downloading role '0', owned by 1.1
 [WARNING]: - openmicroscopy.network was NOT installed successfully: - sorry, 1.1.0 was not found on https://galaxy.ansible.com.
```

Is that meant to be 

```
- src: openmicroscopy.network
  version: 1.1.0
```

Comment (kennethgillen) [reply requirements.yml]: Erm, yes, I guess so. Sorry.

**Comment by kennethgillen** [comment]
@sbesson  - yes, it builds on #39

**Comment by joshmoore** [comment]
Green. Thanks guys. A few things:

 * Can either the description be updated or a README be added to describe what's going on here, when it should be used, by whom, etc.?
 * Are there plans for any testing of these playbooks?
 * What other files are foreseen to live in this directory? e.g. why are `bootstrap` and `postgres` subdirectories and not playbook names?

**Comment by manics** [comment]
`k8s`: A top level directory for Kubernetes-related playbooks
- `bootstrap`, `postgres`: These follow the pattern used for `www` and `web-proxy` of having one directory per server.

**Comment by joshmoore** [comment]
A few other comments, questions, & notes from the recent chat:

 * k8s apps are *not* intended for these directories (see https://github.com/openmicroscopy/kubernetes-apps)
 * From the inventory, what holds `ome-k8s-pg-01` together with the other `[kubernetes-lochy]` machines? Is this PG for use by others outside k8s or will that have networking issues?
 * for testing of the PG setup, the contents of `ome-k8s-pg-01.openmicroscopy.org.yml` (minus the vault usage) should likely come back here
 * How did the pg box get spun up? terraform?

**Comment by manics** [comment]
See https://github.com/kennethgillen/prod-playbooks/pull/1

**Comment by joshmoore** [comment]
Merging to get the 2 PRs with secrets sorted out. Thanks both.

**Pull request merged**